docs/api.rst

```diff
@@ -7,6 +7,7 @@ Information on specific functions, classes, and methods.
    :glob:
 
    api/niworkflows.anat
+   api/niworkflows.cli
```

Comment:
we should address this problem separately and fix master first (sorry, I introduced this with #488)